packages/@okta/vuepress-site/use_cases/event_hooks/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Event Hooks
+excerpt: Use Okta events to drive custom process flows.
+---
+
+# Event Hooks
+
+<ApiLifecycle access="ea" />
+
+## What Are Okta Event Hooks?
+
+Event hooks are outbound calls from Okta, sent when specified events occur in your org. They take the form of HTTPS REST calls to a URL you specify, encapsulating information about the events in JSON objects in the request body. These calls from Okta are meant to be used as triggers for process flows within your own software systems.
```

Comment:
It would be nice if we can mention somewhere that Event Hooks are Okta’s **Webhooks** offering. We need to have one mention of the word "Webhooks" because this will improve the discoverability of the Event Hooks developer docs pages.

Reply:
Let us also include one example use case on when Event Hooks can be used. We can use this example snippet and reword it:

> For example, one can choose to subscribe to user deactivation events in Okta. When this event occur, Okta will notify your endpoint with additional information about that deactivation event. You can write custom code to have this event trigger a downstream workflow such as send an email alert, create a ticket, modify a record in their CRM system, etc.

Reply:
Based on beta feedback and recent customer conversations, we should briefly mention somewhere that in the past, customers had only one option to track events which was to poll for these events using the SystemLog API. So, now with Event Hooks, Okta will push the events only when they occur. This is significantly convenient for the events which are less infrequent yet very urgent.